test/client-stream.js

```diff
@@ -144,56 +144,56 @@ test('stream GET remote destroy', (t) => {
   })
 })
 
-test('stream response resume back pressure and non standard error', (t) => {
```

Comment:
This test was broken before this PR the plan count does not correspond to the actual assertions. Needs to be fixed separatly.